kvariants/bin/sync_dictionaries

```diff
@@ -0,0 +1,9 @@
+set -ex
```

Comment:
could you add a `.sh` extension to the file and a shebang on the top of it:
```suggestion
#!/bin/sh

set -ex
```

Reply:
Also fixed in 6b80181